contacts_theme.libraries.yml

```diff
@@ -22,4 +22,6 @@ action.group:
     state:
       css/contacts/action-group.css: {}
   js:
+    js/util.js: {}
     js/dropdown.js: {}
+    js/collapse.js: {}
```

Comment:
Collapse is a facet library, not related to action-group, should be part of a separate library and also depend on the separate utility library.